src/components/new-request/search.vue

```diff
@@ -598,8 +609,12 @@ export default class Search extends Mixins(CommonMixin) {
   padding: 20px 8px !important;
 }
 
-// set content colour when hovering over list items
+// hide disabled list items
+::v-deep .v-list-item:has(.v-list-item__content.hide-me) {
```

Comment:
This is the first time I've seen this. So `hide-me` is the conditional class you specified in the template. Pretty cool! I'll be keeping this in mind for later work 😄

Reply:
Yes, `:has()` is new-ish.

"apply the following CSS to a v-list-item that has a content element with the hide-me class"

Reply:
I remember you explained it to me once. Awesome to see it in effect!

Reply:
I was being a bit cheeky with the classname, but I think I'll leave it 😆